introduction/quickstart.md

```diff
@@ -0,0 +1,284 @@
+   of transaction data.] After downloading it, query it using its relative or
+   absolute path.
+
+   ```python
+   # Relative path to the same directory from which you started the GlareDB CLI
+   df = con.sql(
+       """
+           SELECT *
+           FROM read_parquet('./f55363e2587849bcb25c057be706c69d-0.parquet');
+       """
+   ).to_pandas()
+
+
+   # Absolute path to the file
+   df = con.sql(
+       """
+           SELECT *
+           FROM read_parquet('/Users/username/Downloads/f55363e2587849bcb25c057be706c69d-0.parquet');
+       """
+   ).to_pandas()
+
+   df
+   ```
+
+7. Query a remote Postgres database using the `read_postgres()` function:
+
+   ```python
+   df = con.sql(
+       """
+           SELECT * FROM read_postgres(
+               'postgresql://demo:demo@pg.demo.glaredb.com:5432/postgres', --connection
+               'public', --schema name
+               'customer' --table name
+           );
+       """
+   ).to_pandas()
+
+   df
+   ```
+
+8. Join between the local Parquet file and the remote Postgres database to
+   get a count of number of sales per NYC borough, and include the names of the
+   NYC borough where the sales took place:
+
+   ```python
+   df = con.sql(
+       """
+           SELECT
+               COUNT(sales.sale_date),
+               lookup.borough_name
+           FROM
+           read_parquet('https://github.com/GlareDB/tutorial_data/raw/main/nyc_sales/sale_year=2022/sale_month=1/f55363e2587849bcb25c057be706c69d-0.parquet') sales
+           JOIN
+               read_postgres(
+                   'postgresql://demo:demo@pg.demo.glaredb.com:5432/postgres',
+                   'public',
+                   'borough_lookup'
+               ) lookup
+           ON sales.borough = lookup.borough_id
+           GROUP BY lookup.borough_name;
+                   );
+       """
+   ).to_pandas()
+
+   df
+   ```
+
+9. Save your data for easy access later using the previous query and `COPY TO`:
+
+   ```python
+   con.sql(
+       """
+           COPY (
+               SELECT
+                   COUNT(sales.sale_date),
+                   lookup.borough_name
+               FROM
+                   read_parquet('https://github.com/GlareDB/tutorial_data/raw/main/nyc_sales/sale_year=2022/sale_month=1/f55363e2587849bcb25c057be706c69d-0.parquet') sales
+               JOIN
+                   read_postgres(
+                       'postgresql://demo:demo@pg.demo.glaredb.com:5432/postgres',
+                       'public',
+                       'borough_lookup'
+                   ) lookup
+               ON sales.borough = lookup.borough_id
+               GROUP BY lookup.borough_name
+           ) TO './my_joined_data.parquet';
+       """
+   )
+
+   df = con.sql(
+       """
+           SELECT *
+           FROM './my_joined_data.parquet';
+       """
+   ).to_pandas()
+
+   df
+   ```
+
+10. If you're using [GlareDB Cloud], create new tables that will
+    persist in Cloud storage using:
```

Comment:
we've not given them any indication of how they _might_ connect to glaredb cloud.

Reply:
I'm just teasing it here to keep it small. If people are interested, they can click the link to sign up for Cloud. When Bullet is out, this will change quite a lot.

Is there something you think would help bridge the gap a bit more without requiring a bunch of additional explanation and steps?

Reply:
I added some next steps which adds another link to Cloud

Reply:
I think we should have either:
- a link to a "sign up to cloud" and "connect to cloud" document (could come next week) 
- a sentence that says "connect to cloud using the connection string provided in the cloud console as an argument to `.connect()" 

But I think we shouldn't just tell people about a thing and then leave them on their own to figure it out.